node/clients/checkout.ts

```diff
@@ -309,6 +311,18 @@ export class Checkout extends JanusClient {
       }
     )
 
+    public changeToAnonymousUser = (orderFormId: string) => {
+      return this.get(this.routes.changeToAnonymousUser(orderFormId), {
+        metric: 'checkout-change-to-anonymous',
+      }).catch((err) => {
+        // This endpoint is expected to return a redirect to
+        // the user, so we can ignore the error if it is a 3xx
+        if (!err.response || /^3..$/.test((err as AxiosError).code ?? '')) {
```

Comment:
Sugestão pra não usarmos regex + explicitar um pouco mais a lógica (assim rola até remover o comentário):
```suggestion
        const canWeIgnoreError = err instanceof AxiosError && err.code >= 300 && err.code < 400
        if (!err.response || canWeIgnoreErrorCode) {
```

Outra opção também é usar o `validateStatus` do axios ao fazer o `this.get(...)`, dessa forma podemos checar se é um `3xx` e não lançar o erro:
https://axios-http.com/docs/handling_errors

Algo como isso:
![Screenshot 2024-02-19 at 09 36 41](https://github.com/vtex-apps/checkout-graphql/assets/20327229/ed840033-c2aa-453b-8573-4b92192feb25)
Mas alterando pra esse case: 
```
{ validateStatus: (status) => status >= 400 }
```
Faz sentido essas opções?